File: BOJ/Java/단계별로_풀어보기/B_02_조건문/BOJ_9498.java
package 단계별로_풀어보기.B_02_조건문;

/**
 * BOJ 9498 : 시험 성적
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class BOJ_9498 {
	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		int A = Integer.parseInt(br.readLine());

		if (A >= 90)
			System.out.println("A");
		else if (A >= 80)
			System.out.println("B");
		else if (A >= 70)
			System.out.println("C");
		else if (A >= 60)
			System.out.println("D");
		else
			System.out.println("F");
		
	}
}

File: BOJ/CPP/단계별로_풀어보기/02_조건문/BOJ_9498.cpp
/*
    BOJ 9498 : 시험 성적

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2021 kdgyun. All rights reserved.
*/

#include <iostream>
using namespace std;

int main(int argc, char const *argv[]) {

    int score;
    cin >> score;

    if (score >= 90) {
        cout << "A";
    } else if (score >= 80) {
        cout << "B";
    } else if (score >= 70) {
        cout << "C";
    } else if (score >= 60) {
        cout << "D";
    } else {
        cout << "F";
    }
    return 0;
}
